src/common/components/MultiToggleSwitch/MultiToggleSwitch.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+
+export type TItem = {
+  label: string
+  count?: number
+}
+
+export type TProps = {
```

Comment:
@ArthurVogelsang I think it would be more descriptive to rename `TProps` to `TMultiToggle`